Mage.Sets/src/mage/cards/s/SurgeNode.java

```diff
@@ -18,23 +17,24 @@
 import mage.counters.CounterType;
 import mage.target.common.TargetArtifactPermanent;
 
+import java.util.UUID;
+
 /**
- *
  * @author Loki
  */
 public final class SurgeNode extends CardImpl {
 
-    public SurgeNode (UUID ownerId, CardSetInfo setInfo) {
-        super(ownerId,setInfo,new CardType[]{CardType.ARTIFACT},"{1}");
-        this.addAbility(new EntersBattlefieldAbility(new AddCountersSourceEffect(new AddCountersSourceEffect(CounterType.CHARGE.createInstance(6))), "{this} gets six charge counters"));
+    public SurgeNode(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{1}");
+        this.addAbility(new EntersBattlefieldAbility(new AddCountersSourceEffect(CounterType.CHARGE.createInstance(6)), "{this} gets six charge counters"));
         Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new AddCountersTargetEffect(CounterType.CHARGE.createInstance()), new GenericManaCost(1));
         ability.addCost(new TapSourceCost());
         ability.addCost(new RemoveCountersSourceCost(CounterType.CHARGE.createInstance()));
         ability.addTarget(new TargetArtifactPermanent());
         this.addAbility(ability);
     }
 
-    public SurgeNode (final SurgeNode card) {
+    public SurgeNode(final SurgeNode card) {
```

Comment:
should be private?

Reply:
Oh the space did mess with my regex. Should be at least protected indeed.

Reply:
I'll add a reflection test for those constructor that should be non-public at some point.